dfget/util/log.go

```diff
@@ -30,69 +27,29 @@ import (
 // DefaultLogTimeFormat defines the timestamp format.
 const DefaultLogTimeFormat = "2006-01-02 15:04:05.000"
 
-// CreateLogger creates a logger.
-func CreateLogger(logPath string, logName string, logLevel string, sign string) (*log.Logger, error) {
```

Comment:
I think that we should retain this function and create a new function to initialize the `logrus` default log instant. And in future, we should share this log package with `dfget`, `dfdaemon`, `supernode`.
`supernode` need to write different log files in different modules, we may use it when refactoring `supernode`.